quay-enterprise/geo-replication.md

```diff
@@ -0,0 +1,36 @@
+# Georeplication of storage in Quay Enterprise
+
+Georeplication allows for a single globally-distributed Quay Enterprise to serve container images from localized storage.
+
+When setup and enabled, all pushes performed will first save their image data to the preferred storage engine for the Quay Enterprise instance. Following push, the image data will be replicated in the background to other storage engines (configurable). Pulling of images will always make use of the closest *avaliable* storage engine, ensuring availability and performance.
+
+**NOTE: Ensure that there is a high availability storage engine (S3, GCS, RADOS, Swift) in each geographic region and that each region can access each storage engine. Local disk storage is not compatible with georeplication at this time.**
```

Comment:
This seems like more than just a "note". Can we restate it as a prereq or some sort of base knowledge?